evm/verify-contract.js

```diff
@@ -47,7 +47,7 @@ async function processCommand(config, chain, options) {
         }
 
         case 'InterchainGovernance': {
-            const InterchainGovernance = require('@axelar-network/axelar-gmp-sdk-solidity/artifacts/contracts/governance/InterchainGovernance.sol/InterchainGovernance.json');
+            const InterchainGovernance = getContractJSON('InterchainGovernance');
```

Comment:
duplicate code for calling getContractJSON for every case. Instead call it once at the top, we already have a variable containing the contractName. the factory can be created generically as well similarly